modules/microshift-data-cleaning-container-images.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-container-images_{context}"]
+= Cleaning all data and keeping the container images
+
+Retain the {microshift-short} container images while cleaning all data by running the `microshift-cleanup-data` script with the `--all` and `--keep-images` arguments.
```

Comment:
Should this be phrased, similar to other sections on this page and throughout the docs, as something the user can do?
```suggestion
You can retain the {microshift-short} container images while cleaning all data by running the `microshift-cleanup-data` script with the `--all` and `--keep-images` arguments.
```